contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -804,37 +804,46 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
     // Validate the report data
     _validateReport(reportContext, report, rs, ss);
 
+    // Verify signatures attached to report
+    _verifySignatures(reportContext, report, rs, ss, rawVs);
```

Comment:
Why it got moved? I have to be missing something, but isn't the main gas saving due to skipping this step if the report is already recorded?